package.json

```diff
@@ -36,7 +36,9 @@
     "lint": "eslint --cache --ext .ts,.tsx src",
     "lint:fix": "npm run lint -- --fix",
     "generate:swagger-schema": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url http://localhost:8000/api/v1/schema/",
-    "generate:form-scaffolds": "ts-node scripts/generateFormScaffolds.ts"
+    "generate:form-scaffolds": "ts-node scripts/generateFormScaffolds.ts",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public"
```

Comment:
`storybook:build`

Reply:
Ah yeah, you're right. Storybook adds it like this by default. 
Will change